Sources/SPMUtility/PkgConfig.swift

```diff
@@ -95,6 +96,10 @@ struct PCFileFinder {
                 return pcFile
             }
         }
+        if PCFileFinder.shouldEmitPkgConfigPathsDiagnostic {
+            PCFileFinder.shouldEmitPkgConfigPathsDiagnostic = false
```

Comment:
We shouldn't reset the state here, otherwise you'll only see this error once when the SwiftPM process is long-lived.

Reply:
Wouldn't that emit the diagnostic with every pc file it tries to look up? Is that OK?

Reply:
Ah, good point. I just realized this isn't going to work well anyway in the long-lived case because we're caching the search paths. Let's take this and track the long lived issue separately. Can you cherry-pick this change to swift-5.1-branch as well?